src/components/vmd-commune-selector.component.ts

```diff
@@ -37,6 +37,7 @@ export class VmdCommuneSelectorComponent extends LitElement {
     @internalProperty() inputHasFocus: boolean = false;
     @property({type: Boolean, attribute: false}) inputModeFixedToText = true;
     @property({type: String, attribute: false}) inputMode: 'numeric'|'text' = 'numeric';
+    @property({ type: String, attribute: false }) optionActiveIndex: number = 0;
```

Comment:
Je me demandais .. y a-t-il une raison particulière pour laquelle tu es passé par une propriété "index selectionné" plutôt que d'ajouter une classe CSS sur l'option sélectionnée (voir de te reposer sur le `[aria-selected='true']`)

Ça éviterait des querySelector un peu compliqués basés sur des `nth-child`, et ça permettrait permettrait d'avoir un language "commun" entre les CSS et le comportement du composant.

Reply:
Sur ma première version, j'utilisais 2 propriétés pour cibler une option possible dans chacune des listes
```ts
@property({ type: String, attribute: false }) communeHighlighted: Commune | undefined;
@property({ type: String, attribute: false }) departementHighlighted: Departement | undefined;
```
Le problème avec ces 2 propriétés qui ciblent 2 listes différentes était de gérer le passage du _highlight_ d'un département à une commune et inversement.

Le passage a une propriété `optionActiveIndex `basée sur l'index de l'option dans la liste `<li>` permet d'alléger considérablement le code.

Le _highlight_ se fait simplement en comparent l'index de la commune/département dans la liste
- pour un département: `aria-selected="${this.optionActiveIndex === index}"`
- pour une commune  . :   `aria-selected="${this.optionActiveIndex === this.departementsAffiches.length+index}"`

La diff. entre les 2 versions est visible sur ce commit https://github.com/CovidTrackerFr/vitemadose-front/pull/108/commits/94bed371606b66eb457b70d66ec00f82bfa5497e

Reply:
yep je comprends bien le fait de ne pas avoir 2 propriétés

Ce que je suggérais, c'était d'en avoir 0 :-) (j'ai l'impression que ça simplifierait encore plus les choses, mais je passe peut-être à coté d'un point de complexité)